File: problem-solving/birthday-cake-problem.js
const prompt = require('prompt-sync')();

const arrSize = prompt();
let candlesHeight = prompt();
candlesHeight = candlesHeight.split(' ');
for (let i = 0; i < arrSize; i++) {
  candlesHeight[i] = parseInt(candlesHeight[i]);
}

const birthdayCakeCandles = candles => {
  let maxHolder = candles[0];
  let maxCounter = 0;
  for (let i = 0; i < candles.length; i++) {
    if (maxHolder === candles[i]) {
      maxCounter++;
    }
    if (maxHolder < candles[i]) {
      maxCounter = 1;
      maxHolder = candles[i];
    }
  }
  return maxCounter;
}

console.log(birthdayCakeCandles(candlesHeight));
// new nlienrfweingft'wel;gmfs;

File: scripts/python/birthday-cake-problem.py
arrSize = int(input())
candlesHeight = input().split(' ')

for i in range(arrSize):
  candlesHeight[i] = int(candlesHeight[i])

def birthdayCakeCandles(candles):
  maxHolder = max(candles)
  maxCount = 0

  for i in range(arrSize):
    if candles[i] == maxHolder:
      maxCount += 1
  
  return maxCount

print(birthdayCakeCandles(candlesHeight))